setup.py

```diff
@@ -3,7 +3,7 @@
 import os
 import setuptools
 
-from src.drozer import meta
+from drozer import meta
```

Comment:
This change was required to enable stdeb to find the right package structure when building deb packages.